index.html

```diff
@@ -14,14 +14,16 @@
 <body>
   <header>
     <nav class="nav">
-      <a class="ic-menu mob" href="#">
-        <img src="./Images/ic_menu.svg" alt="Navigation menu">
-      </a>
+      <div class="hamburger mob">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
       <ul class="menu desk">
-        <li><a href="#home">Hello</a></li>
-        <li><a href="#proj">Portfolio</a></li>
-        <li><a href="#about">About</a></li>
-        <li><a href="#contact">Contact</a></li>
+        <li><a class="menu-link" href="#home">Hello</a></li>
```

Comment:
- Optional
Consider removing this link from your navbar as it doesn't navigate anywhere and is a redundant link.

Reply:
I have removed the redundant link as per your suggestion.